website/markdown/docs/usage/managing-versions.mdx

```diff
@@ -55,7 +55,7 @@ Bundling the version 0.4.0 in the directory /Tuist/.tuist-bin
 ✅ Success: tuist bundled successfully at /Tuist/.tuist-bin
```

Comment:
While you are editing this part of the docs, I think it would be nice to change the `tuist bundle 0.4.0` to `tuist bundle` as well